player/lua/console.lua

```diff
@@ -1519,6 +1519,24 @@ local function command_flags_at_2nd_argument_list(command)
     return flags[command]
 end
 
+local function handle_flags(command, arg_index, flags)
+    for _, cmd in pairs(mp.get_property_native('command-list')) do
```

Comment:
Could we maybe cache command-list? It won't change during runtime.

Reply:
Also you could even make a set of commands with Flags, and search only that, instead of whole thing.

Reply:
I thought of doing this but this loop takes less than 1ms

Reply:
Cached it anyway since it was retrieved in 4 different places, but made it on demand to not slow down the startup time.